cms/djangoapps/contentstore/views/item.py

```diff
@@ -574,7 +574,9 @@ def _duplicate_item(parent_usage_key, duplicate_source_usage_key, user, display_
         duplicate_metadata = {}
         for field in source_item.fields.values():
             if field.scope == Scope.settings and field.is_set_on(source_item):
-                duplicate_metadata[field.name] = field.read_from(source_item)
+                # Do not duplicate 'discussion_id' TNL-1754
+                if field.name != 'discussion_id':
```

Comment:
@catong yes, you are spot on. Also, there is currently a warning in our documentation below that I think we can remove now: http://edx.readthedocs.org/projects/edx-partner-course-staff/en/latest/creating_content/create_discussion.html#create-a-discussion-component
![image](https://cloud.githubusercontent.com/assets/5643426/7379707/ba8453f2-edc4-11e4-8542-833ab731977a.png)